<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-06T13:01:24Z" name="Suite" finished-at="2022-10-06T13:01:47Z" duration-ms="23678">
    <groups>
    </groups>
    <test started-at="2022-10-06T13:01:24Z" name="Bank Manager login Test" finished-at="2022-10-06T13:01:24Z" duration-ms="266">
      <class name="testcases.BankManagerLoginTest">
        <test-method signature="bankManagerLogin()[pri:0, instance:testcases.BankManagerLoginTest@6bff19ff]" started-at="2022-10-06T13:01:24Z" name="bankManagerLogin" finished-at="2022-10-06T13:01:24Z" duration-ms="220" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bankManagerLogin -->
      </class> <!-- testcases.BankManagerLoginTest -->
    </test> <!-- Bank Manager login Test -->
    <test started-at="2022-10-06T13:01:25Z" name="Add Customer Test" finished-at="2022-10-06T13:01:32Z" duration-ms="7506">
      <class name="testcases.AddCustomerTest">
        <test-method signature="addCustomerTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.AddCustomerTest@3b152928]" started-at="2022-10-06T13:01:25Z" name="addCustomerTest" data-provider="dp" finished-at="2022-10-06T13:01:32Z" duration-ms="7286" status="FAIL">
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : Customer added successfully with customer id :6}
  (Session info: chrome=105.0.5195.127): Customer added successfully with customer id :6
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'DESKTOP-K4PK96U', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '16.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [8c8462b2a34493ab1deeb98fa073e90e, screenshot {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 105.0.5195.127, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\Arihan\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:50757}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:50757/devtoo..., se:cdpVersion: 105.0.5195.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 8c8462b2a34493ab1deeb98fa073e90e]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Customer added successfully with customer id :6}
  (Session info: chrome=105.0.5195.127): Customer added successfully with customer id :6
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'DESKTOP-K4PK96U', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '16.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [8c8462b2a34493ab1deeb98fa073e90e, screenshot {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 105.0.5195.127, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\Arihan\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:50757}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:50757/devtoo..., se:cdpVersion: 105.0.5195.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 8c8462b2a34493ab1deeb98fa073e90e
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:131)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
at org.openqa.selenium.remote.RemoteWebDriver.getScreenshotAs(RemoteWebDriver.java:331)
at extentlisteners.ExtentManager.captureScreenshot(ExtentManager.java:48)
at extentlisteners.ExtentListeners.onTestFailure(ExtentListeners.java:49)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:67)
at org.testng.internal.Invoker.runTestListeners(Invoker.java:1389)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:636)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
      </class> <!-- testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
    <test started-at="2022-10-06T13:01:32Z" name="Open Account Test" finished-at="2022-10-06T13:01:47Z" duration-ms="15170">
      <class name="testcases.OpenAccountTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:testcases.OpenAccountTest@644abb8f]" started-at="2022-10-06T13:01:19Z" name="setUp" finished-at="2022-10-06T13:01:24Z" duration-ms="4735" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="openAccountTest(java.lang.String, java.lang.String)[pri:0, instance:testcases.OpenAccountTest@644abb8f]" started-at="2022-10-06T13:01:32Z" name="openAccountTest" data-provider="dp" finished-at="2022-10-06T13:01:37Z" duration-ms="4764" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[smita patil]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rupee]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAccountTest -->
        <test-method signature="openAccountTest(java.lang.String, java.lang.String)[pri:0, instance:testcases.OpenAccountTest@644abb8f]" started-at="2022-10-06T13:01:37Z" name="openAccountTest" data-provider="dp" finished-at="2022-10-06T13:01:47Z" duration-ms="10208" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[anil patil ]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Dollar]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Cannot locate option with text: anil patil 
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'DESKTOP-K4PK96U', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '16.0.1'
Driver info: driver.version: unknown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Cannot locate option with text: anil patil 
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'DESKTOP-K4PK96U', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '16.0.1'
Driver info: driver.version: unknown
at org.testng.Assert.fail(Assert.java:96)
at base.BaseTest.select(BaseTest.java:177)
at testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAccountTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.OpenAccountTest@644abb8f]" started-at="2022-10-06T13:01:47Z" name="tearDown" finished-at="2022-10-06T13:01:48Z" duration-ms="318" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.OpenAccountTest -->
    </test> <!-- Open Account Test -->
  </suite> <!-- Suite -->
</testng-results>
